integration_tests/stacks/lambda-python-function-stack.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+import * as lambda from "@aws-cdk/aws-lambda";
+import { PythonFunction } from "@aws-cdk/aws-lambda-python";
+import * as cdk from "@aws-cdk/core";
+import { Datadog } from "../../src/index";
+
+export class ExampleStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const lambdaPythonFunction = new PythonFunction(this, "HelloHandler", {
+      runtime: lambda.Runtime.PYTHON_3_7,
```

Comment:
Minor nit: Let's use the latest version (Python 3.9) here. Eventually Python 3.7 will get deprecated, so we might as well use the latest version so it lasts longer 🙂